identity-apps-core/apps/authentication-portal/src/main/webapp/ciba-success.jsp

```diff
@@ -0,0 +1,115 @@
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~    http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+--%>
+
+<%@ page import="java.io.File" %>
+<%@ page import="org.wso2.carbon.identity.application.authentication.endpoint.util.AuthenticationEndpointUtil" %>
+<%@ page import="org.owasp.encoder.Encode" %>
+<%@ page import="org.apache.commons.text.StringEscapeUtils" %>
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8"%>
+<%@ taglib prefix="layout" uri="org.wso2.identity.apps.taglibs.layout.controller" %>
+<%@ include file="includes/localize.jsp" %>
+
+<%-- Include tenant context --%>
+<jsp:directive.include file="includes/init-url.jsp"/>
+
+<%-- Branding Preferences --%>
+<jsp:directive.include file="includes/branding-preferences.jsp"/>
+
+<%-- Data for the layout from the page --%>
+<%
+    layoutData.put("isResponsePage", true);
+    layoutData.put("isSuccessResponse", true);
+%>
+
+<!doctype html>
+<html>
+<head>
+    <%-- header --%>
+    <%
+        File headerFile = new File(getServletContext().getRealPath("extensions/header.jsp"));
+        if (headerFile.exists()) {
+    %>
+    <jsp:include page="extensions/header.jsp"/>
+    <% } else { %>
+    <jsp:include page="includes/header.jsp"/>
+    <% } %>
+</head>
+<body class="login-portal layout authentication-portal-layout" onload="loadFunc()">
+    <layout:main layoutName="<%= layout %>" layoutFileRelativePath="<%= layoutFileRelativePath %>" data="<%= layoutData %>" >
+        <layout:component componentName="ProductHeader">
+            <%-- product-title --%>
+            <%
+                File productTitleFile = new File(getServletContext().getRealPath("extensions/product-title.jsp"));
+                if (productTitleFile.exists()) {
+            %>
+                <jsp:include page="extensions/product-title.jsp"/>
+            <% } else { %>
+                <jsp:include page="includes/product-title.jsp"/>
+            <% } %>
+        </layout:component>
+        <layout:component componentName="MainSection">
+            <div class="ui green segment mt-3 attached">
+                <h3 class="ui header text-center slogan-message mt-4 mb-6">
+                    <%=AuthenticationEndpointUtil.i18n(resourceBundle, "successful")%>
+                </h3>
+                <p class="portal-tagline-description">
+                    <%=AuthenticationEndpointUtil.i18n(resourceBundle, "login.success.app")%>
+                    <%= Encode.forHtmlAttribute(request.getParameter("app_name"))%>
+                    <%=AuthenticationEndpointUtil.i18n(resourceBundle, "close.browser")%>
```

Comment:
Let's use the new `i18n` function from localize.jsp here.

Ref: https://github.com/wso2/identity-apps/blob/3c1d844a096f22009a3961fb27b2115a667b9f9e/identity-apps-core/apps/authentication-portal/src/main/webapp/pushAuth.jsp#L192